File: bloaters/long-method/introduce-parameter-object/C#/DateRange.cs
using System;

namespace CSharp
{
    public class DateRange
    {
        public DateTime StartDate { get; }
        public DateTime EndDate { get; }

        public DateRange(DateTime start, DateTime end)
        {
            StartDate = start;
            EndDate = end;
        }

        public TimeSpan TotalDuration()
        {
            return EndDate - StartDate;
        }
    }
}

File: bloaters/long-method/introduce-parameter-object/PHP/dateRange.php
<?php

namespace PHP;

use DateInterval;
use DateTime;

class DateRange
{
    private $startDate;
    private $endDate;

    function __construct(DateTime $start, DateTime $end)
    {
        $this->startDate = $start;
        $this->endDate = $end;
    }

    public function getStartDate(): DateTime
    {
        return $this->startDate;
    }

    public function getEndDate(): DateTime
    {
        return $this->endDate;
    }

    public function totalDuration(): DateInterval
    {
        return $this->startDate->diff($this->endDate);
    }
}
